content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -59,69 +64,26 @@ spec:
   values:
     cni:
       excludeNamespaces:
-       - istio-system
-       - kube-system
-      logLevel: info
-EOF
-$ istioctl install -f istio-cni.yaml
+        - istio-system
+        - kube-system
```

Comment:
nit: why not just make this default in manifests/? Not a blocker for this PR of course